config/settings.py

```diff
@@ -150,19 +150,25 @@
 MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
 MEDIA_URL = '/media/'
 
-EMAIL_HOST = 'smtp.gmail.com'
-EMAIL_PORT = 587
-EMAIL_HOST_USER = 'crazypi6d@gmail.com'
-EMAIL_HOST_PASSWORD = 'django@321'
-EMAIL_USE_TLS = True
-DEFAULT_FROM_EMAIL = 'crazypi6d@gmail.com'
+# authentication stuffs
+from .email_details import *
 
+# TODO:USE YOUR OWN EMAIL SETTINGS FILE 
```

Comment:
This project will be one server , It will have a mail config for it own. We should keep a fake email for testing purpose